app/lib/services.rb

```diff
@@ -12,4 +13,8 @@ def self.publishing_api
   def self.calendars
     @calendars ||= GdsApi::Calendars.new(Plek.new.find('calendars'))
   end
+
+  def self.content_store
```

Comment:
The content store should not be used in publishing apps because it isn't synchronous: there are a number of message queues and caches between the publishing-api and content-store. You can never be sure that you'll get back directly what you just wrote.

If you really need the info in the content-item*, you should use the publishing-api.

Reply:
I agree that normally publishing apps shouldn't talk to the content store, but in this case we need to because it's the only way to get the content_id - we're using YAML files to construct and update content items, rather than a database, and it seems the only way to get information about an existing content item from the Publishing API is to look it up using its content_id...which we don't have.

Reply:
Thanks. We missed `lookup_content_id` when we were looking through the publishing-api docs.